src/generated/resources/data/gtceu/tags/items/electric_pistons.json

```diff
@@ -0,0 +1,17 @@
+{
+  "values": [
+    "gtceu:lv_electric_piston",
+    "gtceu:mv_electric_piston",
+    "gtceu:hv_electric_piston",
+    "gtceu:ev_electric_piston",
+    "gtceu:iv_electric_piston",
+    "gtceu:luv_electric_piston",
+    "gtceu:zpm_electric_piston",
+    "gtceu:uv_electric_piston",
+    "gtceu:uhv_electric_piston",
+    "gtceu:uev_electric_piston",
+    "gtceu:uiv_electric_piston",
+    "gtceu:uxv_electric_piston",
+    "gtceu:opv_electric_piston"
```

Comment:
the high-tier ones need to be marked as optional tag entries so the game won't error.